src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -2928,6 +2936,45 @@ checkNullable:
             Return typeName
         End Function
 
+        Private Function ParseTupleType(openParen As PunctuationSyntax) As TypeSyntax
```

Comment:
This will parse a single-element tuple type such as (Integer) or (x As Integer), which I think is a good thing. We should make sure the rest of the pipeline handles it gracefully.